ui/component/chat/view.jsx

```diff
@@ -16,13 +15,15 @@ import { getTipValues } from 'util/livestream';
 import Slide from '@mui/material/Slide';
 import usePersistedState from 'effects/use-persisted-state';
 import Tooltip from 'component/common/tooltip';
+import { lazyImport } from 'util/lazyImport';
 import './style.scss';
 
 export const VIEW_MODES = {
   CHAT: 'chat',
   SUPERCHAT: 'sc',
 };
 const COMMENT_SCROLL_TIMEOUT = 25;
+const CommentCreate = lazyImport(() => import('component/commentCreate' /* webpackChunkName: "commentCreate" */));
```

Comment:
Just an early heads-up.  Unless intentional, I think we should re-use the `comments` chunk to reduce network traffic.
I think these most likely work together.

<img width="715" alt="image" src="https://user-images.githubusercontent.com/64950861/234280231-48a55185-498c-48a7-b94e-1a376c5625cb.png">